src/class/Prompt.php

```diff
@@ -169,4 +178,23 @@ public function setProjectName($projectName)
 
         return $this;
     }
+
+    /**
+     * @return boolean
+     */
+    public function isInteraction()
+    {
+        return !empty($this->interaction);
+    }
```

Comment:
should I use it here https://github.com/LinkValue/SF2-ReadyToCode/pull/8/files#diff-73f7a38294a8af185aaf3f10dd6f0d70R55 ?